spotbugs/src/main/java/edu/umd/cs/findbugs/detect/BadUseOfReturnValue.java

```diff
@@ -62,8 +62,9 @@ public void sawOpcode(int seen) {
         }
 
         if (seen == Const.INVOKEVIRTUAL && "readLine".equals(getNameConstantOperand())
-                && "()Ljava/lang/String;".equals(getSigConstantOperand()) && getClassConstantOperand().startsWith("java/io")
-                && !"java/io/LineNumberReader".equals(getClassConstantOperand())) {
+                && "()Ljava/lang/String;".equals(getSigConstantOperand()) && 
+                ((getClassConstantOperand().startsWith("java/io") && !"java/io/LineNumberReader".equals(getClassConstantOperand())) ||
+                (getClassConstantOperand().startsWith("javax.imageio.stream")))) {
```

Comment:
The [`getClassConstantOperand()` method always return slashed class name](https://javadoc.io/static/com.github.spotbugs/spotbugs/4.5.0/edu/umd/cs/findbugs/visitclass/DismantleBytecode.html#getClassConstantOperand()), so `getClassConstantOperand().startsWith("javax.imageio.stream")` will not be `true`.

Reply:
@KengoTODA Hi Kengo, I have fixed the problems, could you please check them? Thanks for your kind help.